src/components/Settings/TimeSlotModal/index.js

```diff
@@ -0,0 +1,238 @@
+import React, { useState } from 'react';
+import useStyles from './TimeSlotModal.style';
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  Input,
+} from '@material-ui/core/';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import CalendarIcon from '../../../../src/assets/images/CalendarIcon.svg';
+import DownArrow from '../../../../src/assets/images/DownArrow.svg';
+
+const TimeSlotModal = ({ openModal, setOpenModal, setNotTimeSlot }) => {
+  const isMobile = useMediaQuery('(max-width:420px)');
+  const classes = useStyles({ isMobile });
+  const {
+    root,
+    timeSlotModalWrapper,
+    heading,
+    modalContent,
+    createTitle,
+    titleInput,
+    calendarDateInput,
+    calendarDateWrapper,
+    date_input,
+    calendarInput,
+    weekDaysWrapper,
+    slectedWeekDay,
+    border_right_none,
+    timeSlotsButtons,
+  } = classes;
+
+  const [weekDays, setWeekDays] = useState([
+    {
+      lable: 'Su',
+      selected: false,
+    },
+    {
+      lable: 'M',
+      selected: false,
+    },
+    {
+      lable: 'T',
+      selected: false,
+    },
+    {
+      lable: 'W',
+      selected: false,
+    },
+    {
+      lable: 'Th',
+      selected: false,
+    },
+    {
+      lable: 'F',
+      selected: false,
+    },
+    {
+      lable: 'S',
+      selected: false,
+    },
+  ]);
```

Comment:
can we create a variable outside the component for the initial state?

Reply:
done

Reply:
resolved in commit # https://github.com/femmecubator/femmecubator-ui-repo/pull/150/commits/8c42f0c7febd5499b0d628a6fc4ce0991d76a7dd